.env.development

```diff
@@ -3,3 +3,5 @@ REACT_APP_MAPLIBRE_MAP_STYLE=https://api.maptiler.com/maps/8ecc366c-2d3e-45cf-b3
 REACT_APP_CLIENT_ID=2es98bqdkc6clcneghvbiu997u
 REACT_APP_USERPOOL_ID=eu-central-1_JRz1teeaq
 REACT_APP_REQUESTS_SOURCE=fakeRequestsV2
+REACT_APP_SHOW_REGIONS=0
```

Comment:
So that we have a freedom to enable/disable this feature in DEV/PROD without submitting code.

Say that something gets broken right before Gus has a meeting where he needs to present it.
Or if this feature actually makes the frontend much slower.

Reply:
But you'll need to rebuild the project anyway, so that flag could be easily in JS runtime, I guess.

Reply:
You have to rebuild but you don't need to create/merge any PR requests. Just few clicks through Github's UI to change the value.

I have no clue what JS runtime is.
I just followed what we do with the rest of configuration options like REACT_APP_REQUESTS_SOURCE.